Sources/NIO/Utilities.swift

```diff
@@ -102,23 +109,61 @@ public enum System {
     /// - throws: If an error is encountered while enumerating interfaces.
     @available(*, deprecated, renamed: "enumerateDevices")
     public static func enumerateInterfaces() throws -> [NIONetworkInterface] {
+        var interfaces: [NIONetworkInterface] = []
```

Comment:
Given that this function is deprecated, should we just make it unavailable on Windows?

Reply:
Same as above.